src/DepthCameraSensor.cc

```diff
@@ -636,8 +645,15 @@ double DepthCameraSensor::NearClip() const
 
 //////////////////////////////////////////////////
 bool DepthCameraSensor::HasConnections() const
+{
+  return this->HasImageConnections() ||
+      this->HasInfoConnections();
+}
+
+//////////////////////////////////////////////////
+bool DepthCameraSensor::HasImageConnections() const
 {
   return (this->dataPtr->pub && this->dataPtr->pub.HasConnections()) ||
-      (this->dataPtr->pointPub && this->dataPtr->pointPub.HasConnections()) ||
-      this->dataPtr->imageEvent.ConnectionCount() > 0u;
+         (this->dataPtr->pointPub && this->dataPtr->pointPub.HasConnections())
+         || this->dataPtr->imageEvent.ConnectionCount() > 0u;
```

Comment:
to be consistent with RGBD camera, separate out the check for points to its own `HasPointConnections`? 

also for consistency with RGBD camera, maybe this function should also be named `HasDepthConnections`

Reply:
Good point!